integration-tests/tests/specs/auth/logout.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from '@playwright/test';
+
+import { test } from '../../fixtures/auth.fixture';
+import { AuthPage } from '../../pages/auth.page';
+
+const AUTH_COOKIE_NAMES = ['access_token', 'refresh_token'];
+
+test.describe('Logout Flow', () => {
+    let authPage: AuthPage;
+
+    test.beforeEach(({ page }) => {
+        authPage = new AuthPage(page);
+    });
+
+    test('should logout from the account menu', async ({ page, testAccount }) => {
+        await authPage.createAccount(testAccount);
+        await page.waitForLoadState('networkidle');
+
+        await authPage.logout();
+
+        await expect(page).toHaveURL(/\/logout$/);
+        await expect(page.getByText('You have been logged out')).toBeVisible();
+
+        const cookies = await page.context().cookies();
+        const authCookies = cookies.filter((cookie) => AUTH_COOKIE_NAMES.includes(cookie.name));
```

Comment:
Do we want to check whether those cookies are set before the logout? 

Testing for absence is always dangerous. You might end up with a test that does nothing but is still green.

Reply:
Totally agreed but prefer to decouple this since I think it's a pre-existing issue and I don't have bandwidth this sec - made an issue!